.github/workflows/tests.yml

```diff
@@ -19,19 +19,13 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Set up Ruby
-        uses: ruby/setup-ruby@a699edbce608a2c128dedad88e3b6a0e28687b3c
+        uses: ruby/setup-ruby@b161cb92f0e2f28d992f04f38fff204d84132c47
         with:
-          ruby-version: 2.6.6
-
-      - name: Install gems
-        run: |
-          gem install rubocop -v 1.7.0
-          gem install rubocop-minitest -v 0.11.0
-          gem install rubocop-performance -v 1.10.1
-          gem install rubocop-rails -v 2.9.1
+          ruby-version: .ruby-version
+          bundler-cache: true
 
       - name: Run Rubocop
-        run: rubocop --except Metrics
+        run: bundle exec rubocop --except Metrics
```

Comment:
So I deliberately wasn't using bundler here but I think that was because the tooling libraries (e.g. test runner) don't use Bundler as it makes things too slow. For the website, it makes sense I think.

Reply:
Reasoning to not use makes sense.

I have wondered about putting Rubocop in it's own Rubygems group and installing just that.

Reply:
Reverted to original.

Reply:
That's not a bad shout. For the test-runner stuff, I mainly discovered that running the tests with `rake test` takes 0.5s, and with `bundle exec rake test` it takes 2s. When the "product" is running tests fast, that was a ridiculous difference. Then I think these all just copied from that world.